src/main/scala/mesosphere/marathon/core/task/Task.scala

```diff
@@ -161,20 +161,16 @@ object Task {
 
     def attempt(taskId: String): Option[Long] = {
       taskId match {
+        case ResidentTaskIdWithInstanceIdRegex(runSpecId, prefix, uuid, container, attempt) => Some(attempt.toLong)
```

Comment:
`(_...)`?

Reply:
It is less-readable. I intentionally kept it the way it was and still is.